internal/api/installation_test.go

```diff
@@ -468,15 +468,6 @@ func TestUpgradeInstallation(t *testing.T) {
 		require.Equal(t, model.InstallationStateUpgradeRequested, installation1.State)
 	})
 
-	t.Run("while stable, to invalid version", func(t *testing.T) {
```

Comment:
Can we replace this test with one that asserts the requested version is written to the installation record?

Reply:
Sure thing!